### PR TITLE
[Fix] enters in schema field

### PR DESCRIPTION
**Pull request by Abrams27**
closes https://github.com/TouK/nussknacker/issues/862
Screenshot from 2020-05-19 13-10-14
Screenshot from 2020-05-19 13-11-18

### CODE REVIEW

File: ui/client/components/graph/node-modal/editors/expression/Editor.ts [hunk 1]
```diff
@@ -62,6 +63,7 @@ export enum EditorType {
   DURATION_EDITOR = "DurationParameterEditor",
   PERIOD_EDITOR = "PeriodParameterEditor",
   CRON_EDITOR = "CronParameterEditor",
+  TEXTAREA_PARAMETER_EDITOR = "TextareaParameterEditor"
```
Comment (lciolecki) [inline ui/client/components/graph/node-modal/editors/expression/Editor.ts:66]: missing comma

File: ui/client/components/graph/node-modal/editors/expression/TextareaEditor.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+const stringPattern = /(^'.*'$)|(^".*"$)/
+
+const parseable = (expressionObj) => {
+  const expression = expressionObj.expression
+  const language = expressionObj.language
+  return stringPattern.test(expression) && language === "spel"
+}
+
+TextareaEditor.switchableTo = (expressionObj) => parseable(expressionObj)
+TextareaEditor.switchableToHint = () => i18next.t("editors.string.switchableToHint", "Switch to basic mode")
```
Comment (lciolecki) [inline ui/client/components/graph/node-modal/editors/expression/TextareaEditor.tsx:40]: `editors.textarea`

**Comment by lciolecki** [comment]
Please add info in changelog.

File: ui/client/components/graph/node-modal/editors/expression/TextareaEditor.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import Textarea from "../field/Textarea"
+
+type Props = {
+    expressionObj: $TodoType,
+    onValueChange: Function,
+    className: string,
+    formatter: Formatter,
+}
+
+const TextareaEditor: Editor<Props> = (props: Props) => {
```
Comment (mproch) [inline ui/client/components/graph/node-modal/editors/expression/TextareaEditor.tsx:14]: can we extract common stuff with text input? switchableTo, (non)switchableToHint are the same?

**Pull request merged**